back/spec/models/application_record_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe ApplicationRecord, type: :model do
+  subject(:user) { create(:user) }
+
+  describe '#valid_attribute?' do
```

Comment:
I was thinking also of adding a case where `valid_attribute?` is called twice, but not blocking for me :)